### PR TITLE
fix some bad indentations

### PR DESCRIPTION
**Pull request by a14n**

### CODE REVIEW

File: packages/flutter_tools/lib/src/commands/update_packages.dart [hunk 1]
```diff
@@ -640,7 +640,7 @@ class PubspecYaml {
         // place to insert our transitive dependencies.
         if (section == Section.dependencies)
           endOfDirectDependencies = output.length;
-          endOfDevDependencies = output.length;
+        endOfDevDependencies = output.length;
```
Comment (a14n) [inline packages/flutter_tools/lib/src/commands/update_packages.dart]: This one looked strange. Not sure if this line should be inside or outside of the if section.

Comment (dnfield) [reply packages/flutter_tools/lib/src/commands/update_packages.dart]: I suspect this line should just be removed. It looks like a copy paste error and is almost certainly wrong.

Comment (dnfield) [reply packages/flutter_tools/lib/src/commands/update_packages.dart]: /cc @jonahwilliams blame suggests you made this change.  Can you confirm this line should be removed?

**Comment by dnfield** [comment]
LGTM modulo that one line that I'm pretty sure needs to be removed.

Can I ask how you found these?  Do you have a script or regex or something that is checking it?  Is it something we could potentially integrate into tooling for the repo?

File: packages/flutter_tools/lib/src/commands/update_packages.dart (hunk 1 above)
Comment (dnfield) [reply packages/flutter_tools/lib/src/commands/update_packages.dart]: If you look a couple lines down it gets set if you're in the expected section.  It looks like if it does work it's probably got bugs (counts the dependencies for the wrong section(s)).

Comment (a14n) [reply packages/flutter_tools/lib/src/commands/update_packages.dart]: Line removed and tests still work(locally)

**Comment by a14n** [comment]
>Do you have a script or regex or something that is checking it?

I made an analyzer plugin to check the formatting. Formatting issues are displayed in VSCode and I can fix them. It's in an early stage and need more work (especially on the tests side - I miss some examples about how to testing an analyzer plugin).

>Is it something we could potentially integrate into tooling for the repo?

I have that in mind and I'd like to add it. Before that I saw strange things with flutter command line that I need to explore (but I'm not sure to be able to fix that on my own).

**Comment by DanTup** [comment]
> I made an analyzer plugin to check the formatting. Formatting issues are displayed in VSCode and I can fix them.

Out of interest, how involved was this? (making the server load/run your plugin, rather than actually implementing it).

**Comment by a14n** [comment]
>Out of interest, how involved was this? (making the server load/run your plugin, rather than actually implementing it).

It works quite well. I only had to add my plugin as dev dependency and add a section in `analysis_options.yaml` to register my plugin. After a restart the new warnings appeared. \o/

BTW I noticed a strange behaviour with fixes. All fixes from the same file was displayed on every lints (after a right click on a lint I was seen a huge list of fixes). But it was perhaps an issue on my side... For now I removed quick fixes.

**Comment by dnfield** [comment]
That's super cool.  I'm really excited to see what comes of it - formatting and linting the style of this repo is definitely a pain point.  If you're willing to put up what you have I'm sure more people would be interested in using/contributing to it.

Thanks again for all the work you do on this!

**Pull request merged**